jenkins/spark-tests.sh

```diff
@@ -242,12 +242,30 @@ run_avro_tests() {
     ./run_pyspark_from_build.sh -k avro
 }
 
+rapids_shuffle_smoke_test() {
+    echo "Run rapids_shuffle_smoke_test..."
+
+    # using MULTITHREADED shuffle
+    PYSP_TEST_spark_rapids_shuffle_mode=MULTITHREADED \
+    PYSP_TEST_spark_rapids_shuffle_multiThreaded_writer_threads=2 \
+    PYSP_TEST_spark_rapids_shuffle_multiThreaded_reader_threads=2 \
+    PYSP_TEST_spark_cores_max=2 \
+    PYSP_TEST_spark_executor_cores=1 \
```

Comment:
this set of setting is not ideal in my opinion, we run integration tests in  a bunch of environments, some are actually distributed with different setups, like the yarn cluster, this seems like a set of configs specifically for local mode, unless these are being overwritten in those different environments?

Reply:
Agree. @nartal1 do you know what are all the environments these tests are running on? The main reason to set all of this is if we are going to share a GPU, like the pre-merge tests do. 

If the tests are running in a distributed environment, and you get 1 executor/GPU, the cores and memory configs should go away.

Reply:
I think we will be running this as part of nightly integration tests. I will remove the configs mentioned above and update the PR.